sdk/communication/azure-communication-identity/azure/communication/identity/_communication_identity_client.py

```diff
@@ -154,15 +180,35 @@ def get_token(
         :type user: ~azure.communication.identity.CommunicationUserIdentifier
         :param scopes: List of scopes to be added to the token.
         :type scopes: list[str or ~azure.communication.identity.CommunicationTokenScope]
+        :param token_expires_after: Custom validity period of the Communication Identity access token within <60,1440> minutes range. If not provided, the default value of 1440 minutes (24 hours) will be used.
+        :type token_expires_after: timedelta
         :return: AccessToken
         :rtype: ~azure.core.credentials.AccessToken
         """
         api_version = kwargs.pop("api_version", self._api_version)
+        
+        expires_after_in_minutes = 0
+        
+        if (token_expires_after is not None):
+            
+            # timedelta counts seconds that sum up to one day and assigns them to days property, remainder is assigned to seconds property
+            # i.e. when reaching 86400 seconds, value for days property will be 1, value for seconds property will be 0
+            expires_after_in_minutes = int((token_expires_after.days * self.MAX_TOKEN_VALIDITY_IN_SECONDS + token_expires_after.seconds) / 60)
```

Comment:
```suggestion
            expires_after_in_minutes = int(token_expires_after.total_seconds / 60)
```
why did not we use `total_seconds()` directly?

Reply:
fixed